
Starting evaluation
/home/robotlabx/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
Average eval rmse: 623101.9128714809 in 37.10427713394165 seconds
best_prev_rmse 623101.9128714809
<class 'float'>
Traceback (most recent call last):
  File "enet_sanity.py", line 441, in <module>
    {"best previous evaluaion rmse": best_prev_rmse})
  File "/home/robotlabx/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/wandb/sdk/wandb_run.py", line 1323, in log
    if self.history._step > step:
TypeError: '>' not supported between instances of 'int' and 'dict'